tests/resources/test-types.json

```diff
@@ -1045,7 +1117,7 @@
     "nextTestTypesOrCategories": [
       {
         "id": "38",
-        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","39"],
+        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","39","41"],
```

Comment:
you should also add the category's id which is 40

Reply:
I dont see 11,12,13 added but categories -14, 15,16 added. Same is the case with 20, 26 which are not added, but (21,22) and (27,28) added here. Kindly verify and let me know if both 92 and 93 is required to be added here.

Reply:
Done.